styles.css

```diff
@@ -1629,4 +1826,59 @@ button {
   .nav-menu .icon {
     display: none;
   }
+
+  .modal {
+    max-height: 100%;
+    max-width: 70%;
+  }
```

Comment:
Great job team 🤝

- [ ] You may kindly fix the responsiveness of your design as some content seems to be overlapping other pieces of the design.
![deaktop overlap](https://user-images.githubusercontent.com/63917725/180295203-1c780730-ccf9-469e-b6c1-4a7a670f3a96.PNG)

Reply:
- [ ] Corrected